api/src/client/dynamics/elevator-safety/types.ts

```diff
@@ -9,6 +9,14 @@ export interface ElevatorSafetyRegistrationClient {
   ) => Promise<ElevatorRegistration[]>;
 }
 
+export interface ElevatorSafetyViolationsClient {
+  getViolationsForPropertyInterest: (
+    accessToken: string,
+    address: string,
+    municipalityId: string
+  ) => Promise<boolean>;
```

Comment:
Right now this is just a `Promise<boolean>` because we only care about whether we found any violations for this address. Future iterations will alter this return type to be more meaningful